third_party/amd/lib/TritonAMDGPUTransforms/DecomposeConversions.cpp

```diff
@@ -49,11 +49,22 @@ class TritonAMDGPUDecomposeConversionsPass
             dstDotOp.getParent() == srcMfmaEncoding)
           return;
       }
+      auto srcOrder = triton::gpu::getOrder(srcEncoding);
```

Comment:
Can we run `add_reduce_data_duplication` and delete this file ? Seems exact replica of `createReduceDataDuplicationPass`

Reply:
Didn't notice this. Will do.